packages/cli/src/cmds/beacon/initBeaconState.ts

```diff
@@ -104,7 +104,17 @@ export async function initBeaconState(
     if (!args.weakSubjectivityServerUrl) {
       throw Error(`Must set arg --weakSubjectivityServerUrl for network ${args.network}`);
     }
-    logger.info("Fetching weak subjectivity state", {weakSubjectivityServerUrl: args.weakSubjectivityServerUrl});
+    try {
+      // Validate the weakSubjectivityServerUrl and only log the origin to mask the
+      // username password credentials
+      const wssUrl = new URL(args.weakSubjectivityServerUrl);
+      logger.info("Fetching weak subjectivity state from", {
```

Comment:
```suggestion
      logger.info("Fetching weak subjectivity state", {
```

Reply:
@wemeetagain I added *from* looking at the final generated log: `info: Fetching weak subjectivity state from weakSubjectivityServerUrl=https://eth2-beacon-` which reads good with from. Let me know what you think. :slightly_smiling_face:

Reply:
You shouldn't assume how the URL renders, we could change that in the future